app/views/shared/_received-booking-card.html.erb

```diff
@@ -0,0 +1,20 @@
+<% @my_listings.each do |listing| %>
+  <% if listing.bookings.nil? %>
+    <p>Wow, such empty! 👶</p>
```

Comment:
nice to add this user friendly message, maybe rephrase this sentence?

Reply:
Yes, we can add "You have no bookings"